pkg/controllers/clusterresourceplacement/controller_test.go

```diff
@@ -2017,6 +2079,57 @@ func TestGetOrCreateClusterResourceSnapshot_failure(t *testing.T) {
 	}
 }
 
+func TestSplitSelectedResources(t *testing.T) {
+	serviceResourceContent := *serviceResourceContentForTest(t)
```

Comment:
could you please write a comment to indicate the size of content of the resource? easier for reader to understand what you're testing.